crates/core/src/db/queries/wallet.rs

```diff
@@ -8,14 +8,14 @@ use diesel::{
 
 use crate::{
     db::{
-        models::{CollectedCollection, CreatedCollection, WalletActivity},
+        models::{CollectedCollection, CreatedCollection, Offer, WalletActivity},
         Connection,
     },
     error::prelude::*,
 };
 
 const ACTIVITES_QUERY: &str = r"
-SELECT listings.id as id, metadata, auction_house, price, auction_house, created_at, marketplace_program,
```

Comment:
Why the remove of auction_house?

Reply:
Because it was written twice in the query.